pepper-apis/dsm-core/src/main/resources/liquibase/juniper-ourheart/PEPPER-1057-our_heart_setup.xml

```diff
@@ -0,0 +1,110 @@
+        <insert tableName="kit_return_information">
+            <column name="return_address_name" value="Broad Institute"/>
+            <column name="return_address_street1" value="Attn. Broad Genomics"/>
+            <column name="return_address_street2" value="320 Charles St. – Lab 132"/>
+            <column name="return_address_city" value="Cambridge"/>
+            <column name="return_address_state" value="MA"/>
+            <column name="return_address_zip" value="02141"/>
+            <column name="return_address_country" value="US"/>
+            <column name="return_address_phone" value=""/>
+        </insert>
+    </changeSet>
+    <changeSet id="our-heart-ddp-kit-request-settings-insert" author="pegah">
+        <insert tableName="ddp_kit_request_settings">
+            <column name="ddp_instance_id" valueComputed="(SELECT ddp_instance_id from ddp_instance WHERE study_guid = 'juniper-ourheart')"/>
+            <column name="kit_type_id" valueComputed="(SELECT kit_type_id from kit_type WHERE kit_type_name = 'SALIVA')"/>
+            <column name="kit_return_id" valueComputed="(SELECT kit_return_id FROM kit_return_information WHERE return_address_street2 = '320 Charles St. – Lab 132')"/>
+            <column name="carrier_service_to_id" valueComputed="(SELECT carrier_service_id FROM carrier_service WHERE service = 'FEDEX_EXPRESS_SAVER' )"/>
+            <column name="kit_dimension_id" valueComputed="(SELECT kit_dimension_id from kit_dimension WHERE kit_description like 'juniper-ourheart saliva kit')"/>
+            <column name="kit_label_length" value="14"/>
+        </insert>
+    </changeSet>
+    <changeSet id="our-heart-ddp-group-roles" author="pegah">
+        <insert tableName="ddp_group_role">
+            <column name="group_id" valueComputed="(SELECT group_id FROM ddp_group WHERE name='juniper-ourheart')"/>
+            <column name="role_id" valueComputed="(SELECT role_id FROM access_role WHERE name='study_user_admin')"/>
+        </insert>
+        <insert tableName="ddp_group_role">
+            <column name="group_id" valueComputed="(SELECT group_id FROM ddp_group WHERE name='juniper-ourheart')"/>
+            <column name="role_id" valueComputed="(SELECT role_id FROM access_role WHERE name='kit_shipping')"/>
+            <column name="admin_role_id" valueComputed="(SELECT role_id FROM access_role WHERE name='study_user_admin')"/>
+        </insert>
+        <insert tableName="ddp_group_role">
+            <column name="group_id" valueComputed="(SELECT group_id FROM ddp_group WHERE name='juniper-ourheart')"/>
+            <column name="role_id" valueComputed="(SELECT role_id FROM access_role WHERE name='kit_deactivation')"/>
+            <column name="admin_role_id" valueComputed="(SELECT role_id FROM access_role WHERE name='study_user_admin')"/>
+        </insert>
+    </changeSet>
+    <changeSet id="juniper-ourheart-pepper-admin" author="pegah">
```

Comment:
This should not be part of Liquibase. It is designed to be done by hand, and if we want to revoke permission we would have to do a product release.

Reply:
Removed!
I misunderstood our slacks from yesterday, sorry!